seqr/fixtures/1kg_project.json

```diff
@@ -1237,6 +1236,44 @@
         "loaded_date": "2017-02-05T06:42:55.397Z"
     }
 },
+{
+    "model": "seqr.sample",
+    "pk": 152,
+    "fields": {
+        "guid": "S000152_na19675_d2",
+        "created_date": "2017-02-05T06:42:55.397Z",
+        "created_by": null,
+        "last_modified_date": "2017-03-13T09:07:49.744Z",
+        "individual": 1,
+        "sample_type": "RNA",
+        "dataset_type": "VARIANTS",
+        "sample_id": "NA19675_D2",
+        "is_active": true,
+        "elasticsearch_index":null,
+        "tissue_type": "M",
+        "data_source": "muscle_samples.tsv.gz",
+        "loaded_date": "2017-02-05T06:42:55.397Z"
+    }
+},
+{
+    "model": "seqr.sample",
+    "pk": 153,
+    "fields": {
+        "guid": "S000153_na19675_d2",
```

Comment:
Add this sample and a TPM entry afterward to test if the TPM data is deleted when adding data for a different tissue.